otcextensions/tests/unit/sdk/cbr/v3/test_member.py

```diff
@@ -0,0 +1,63 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
```

Comment:
You forgot to add 
```sh
#!/usr/bin/env python3
```

Reply:
all of our test  files do not have the shebang line registered. So I don't see a (real) issue here.